scripts/core/config.ts

```diff
@@ -1,7 +1,7 @@
 import _ from 'lodash';
 
 export function isMediaEditable(config) {
-    return !!_.get(config, 'features.editFeaturedImage');
+    return _.get(config, 'features.editFeaturedImage', true) !== false;
```

Comment:
Is there a reason you use `!== false` instead of `=== true`?

Reply:
to make it enabled when it's not set

Reply:
actually, I'm not sure now